e2e/cp_test.go

```diff
@@ -3346,3 +3347,73 @@ func TestCopyRawModeAllowDestinationWithoutPrefix(t *testing.T) {
 		t.Errorf("testfile*.txt not exist in S3 bucket %v\n", dst)
 	}
 }
+
+func TestCopyFalseCredentialsExitCode1(t *testing.T) {
```

Comment:
```suggestion
func TestCopyExpectExitCode1OnEmptyCredentials(t *testing.T) {
```

Reply:
removed credentials test because, it is not related with our case. Same test case works same with `master` branch.